### PR TITLE
Add draft of abstract

### PR DESCRIPTION
**Pull request by pinetree408**
Abstract Draft입니다. 총 161 word 나왔구요, 최대 200 단어 제한입니다.
저희끼리의 내용 컨펌도 안난 상태이기때문에 아주 러프하게 작성했습니다.
대략 150이상 왔다 갔다하면 충분할 것 같은데 보시고 논의 하시죠!
내용적인 부분에서, 문법적인 부분까지 또는 문장단위 review도 다 환영합니다.

@junhwi @byou-kaist

### CODE REVIEW

**Review by byou-kaist — COMMENTED** [review]
we discussed a proper use of JWT ~를 뒷 문장과 같이 proper usage로 바꾸는것이 좋을것 같아요

File: vulnerability-jwt/template.tex [hunk 1]
```diff
@@ -87,7 +87,8 @@
 
 % Abstract (Do not use inserted blank lines, i.e. \\) 
 \abstract{
-% TODO
+Recent trends made Token based authentication become popular. Token based authentication complements many problems of Server based authentication. However, recently reported issues have shown that token based authentication is still less mature in terms of security. So we tried to analyze the problem. Before analyzing, We compared server based authentication with token based authentication, and examined how token based authentication solves the problems of existing methods. In order to analyze the problem, we targeted JWT which is highly usable among Token based authentication methods. We conducted a field study to analyze various libraries supporting JWT and diagnosed the problem. Based on the results of the diagnosis, We categorized vulnerabilities of JWT into 3 category(Modifiable JOSE header, Bruteforce attack, Replay attack). We presented and verified attack scenarios for each category. Finally, we discussed a proper usage of JWT based on the previous two studies. The proper usage of JWT is claiming someone is authenticated from another service in cryptographic reliable way.
```
Comment (junhwi) [inline vulnerability-jwt/template.tex]: Token based authentication complements many problems of `server` based authentication. `JWT is the latest standard of token using JSON format.` However, recently reported issues have shown that `JWT` is still less mature in terms of security.

Comment (junhwi) [inline vulnerability-jwt/template.tex]: Recent trends made Token based authentication become popular. 삭제해도 되지 않을까요?

**Review by byou-kaist — COMMENTED** [review]
어느정도 잘 정리되서 마무리하면 될 것 같습니다. 나머지는 본문이나 conclusion 추가되는 것들 보면서 abstract에 반영시키기만 하면 될것 같네요.

File: vulnerability-jwt/template.tex (hunk 1 above)
Comment (junhwi) [inline vulnerability-jwt/template.tex]: `In this paper`, `we` compared server based authentication with token based authentication, and examined how token based authentication solves the problems of existing methods. `After that, we examined JWT in more detail.` 하고
In order to analyze the problem, we targeted JWT which is highly usable among Token based authentication methods.는 지워도 될듯합니다

Comment (junhwi) [inline vulnerability-jwt/template.tex]: We conducted a field study to analyze `the current status of` various libraries supporting JWT `in security perspective`. Here, we propose 3 kinds of potential vulnerability of JWT, modifiable JOSE header, weakness to brute force attack and replay attack. We presented and verified attack scenarios for each category. Also, we suggest a proper usage of JWT based on the previous two studies. The proper usage of JWT is claiming authentication between multiple parties in cryptographic reliable way.

Comment (junhwi) [reply vulnerability-jwt/template.tex]: 뒷부분 다시 고쳤습니다 한번 확인해보세요

**Comment by pinetree408** [comment]
@junhwi @byou-kaist 두분 리뷰다 반영했습니다. 확인 부탁드려요~

**Review by junhwi — APPROVED** [review]

**Pull request merged**